### ISSUE TITLE
incorrect line number and column for expressions in f-strings

### ISSUE THREAD

**Issue by jwkvam** [PyCQA/pylint#1595]
### Steps to reproduce
1. Create the following python code.
```
a = 1
s = f'hello = {  c[0]}'
```
2. Run `pylint -E`
```
E:  1, 3: Undefined variable 'c' (undefined-variable)
```

### Current behavior
The reported line number and column seem to be in reference to the braces in the f-string. Notice that there are two spaces before the reference to the nonexistent `c`, thus column 3 is reported.

### Expected behavior
I want the line number and column within the file. It's difficult to find the offending code otherwise.

### pylint --version output
```
pylint 1.7.2,
astroid 1.5.3
Python 3.6.1 | packaged by conda-forge | (default, May 23 2017, 14:31:56)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)]
```

**Comment by rogalski** [comment]
#1373 have to be mentioned here. Both have same root cause. We need to investigate why original fix have not helped.

**Comment by PCManticore** [comment]
The original fix is just for FormattedValues, not for nested attributes of the node.

**Comment by brycepg** [comment]
This issue appears to be fixed

**Comment by jwkvam** [comment]
Awesome, I tested this and it seems to work now, I'll close. Thanks to whomever fixed it :)

**Comment by brycepg** [comment]
Hmm I'm able to reproduce this again on master

    Using config file /home/bguinta/pm5-dev/pylintrc
    __main__.py 2.0.0 bd38d67ab5cba0d850c54753677c8f5f40b38004
    astroid 2.0.0 3fae32f99c7f432a1281b151ea17e163a2e8dc47
    Python 3.6.1 |Continuum Analytics, Inc.| (default, May 11 2017, 13:09:58) 
    [GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]

**Comment by brycepg** [comment]
I'm going to at least add a test for this issue for reproducibility on another machine

**Comment by gyermolenko** [comment]
Did not reproduce with pylint from master
```console
$ pylint --version
pylint 2.4.0-dev0
astroid 2.3.0
Python 3.7.4 (default, Aug 11 2019, 19:46:18) 
[GCC 8.3.0]

$ cat fstr.py 
a = 1
s = f'hello = {  c[0]}'

$ pylint -E fstr.py 
************* Module fstr
fstr.py:2:17: E0602: Undefined variable 'c' (undefined-variable)
```

**Comment by PCManticore** [comment]
This seems to reproduce with 3.6.1 only, at this time I think it's safe to close this one.
